covidX/settings.py

```diff
@@ -96,11 +96,13 @@
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
 
-if os.getenv("GAE_APPLICATION", None):
+if GAE_APPLICATION := os.getenv("GAE_APPLICATION", False) and (
```

Comment:
Codacy found an issue: [os.getenv default type is builtins.bool. Expected str or None.](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6289626)